### DISCUSSION TITLE
HTTP 508 error "INFINITE_LOOP_DETECTED"

### DISCUSSION THREAD

**Discussion by Neodark7** [vercel/community#862] [Help] [answered]
Hi ! i'm getting this error when fetching the projects list of my team using the Vercel API (same with Postman and Curl). any idea ?

**Comment by Neodark7** [comment, marked as answer]
my bad, i'm dumb, it was an error from my request url i used "&" to pass the teamId query parameter instead of just "?".
Having this specific response for this error is pretty weird tho 🤔

**Reply by swarnava** [reply]
HI, What was the error?

**Reply by Neodark7** [reply]
it's in the issue title, i got an HTTP 508 error with "INFINITE_LOOP_DETECTED" in the response body
